config/jobs/GoogleCloudPlatform/k8s-cluster-bundle/k8s-cluster-bundle.yaml

```diff
@@ -0,0 +1,44 @@
+presubmits:
+  GoogleCloudPlatform/k8s-cluster-bundle:
+  - name: gcp-k8s-cluster-bundle-presubmit
+    always_run: true         # Run for every PR, or only when requested.
+    branches:
+    - master
```

Comment:
Unless we are intentionally avoiding running the jobs on non-`master` branches for some reason we should get rid of the `branches` sections. If we are intentionally only running the job on `master` please include a comment explaining why.

Reply:
Removed branches sections.